src/components/SubList.jsx

```diff
@@ -0,0 +1,32 @@
+import { ListItem } from './ListItem';
+
+function SubList({ category, listToken, urgency, searchQuery }) {
+	const urgencyTitle = Object.keys(category)[0];
+	const urgencyArray = Object.values(category);
+
+	return (
+		<>
+			<h2>{urgencyArray[0].length === 0 ? null : urgencyTitle}</h2>
```

Comment:
Is this the line that isn't working as you want it to? From what I caught during our sync, it sounds like y'all only want to display the urgency title if there is actually content in the associated sublist (which makes sense to me).

I think you're close to being able to get that functionality working - have been messing around with the code a little and broke it further 😅 but will take another stab at it this afternoon!

Reply:
we don't want to see urgency headers if the item we are looking for is not there
![Screenshot 2022-11-19 at 2 21 45 PM](https://user-images.githubusercontent.com/85965595/202867959-4a69aad5-6a63-4f6c-bf2e-efb42bb2b6b8.png)